etc/sinks/redis.json

```diff
@@ -0,0 +1,133 @@
+{
+	"about": {
+		"trial": false,
+		"author": {
+			"name": "DengFeng HE",
+			"email": "contact@emqx.io",
```

Comment:
Could you update with your own email, company and website or just leave them empty

Reply:
ok